tools/size-generator.mjs

```
@@ -0,0 +1,28 @@
+import { generateFileSizeReport } from "@jsenv/file-size-impact";
+
+new URL("./", import.meta.url);
```

Not sure, is this intended (maybe to throw an error for some weird URLs) or is this a leftover?

---

Ha. You found my easter egg! Seriously, great catch : rocket:!

---

The easter bunny came early this year 😆